flac.md

```diff
@@ -393,6 +394,36 @@ Value           | Description
 0b1010          | mid/side stereo: channel 0 is the mid(average) channel, channel 1 is the side(difference) channel
 0b1011 - 0b1111 | reserved
 
+In case different channels than those in the table above are coded, it is recommended to add a WAVEFORMATEXTENSIBLE_CHANNEL_MASK to a VORBIS_COMMENT metadata block to describe which channels are stored. This channel mask consists of flag bits indicating which channels are present, stored in a hexadecimal representation preceded by 0x. The flags only signal which channels are present, not in which order, so in case a file has to be encoded in which channels are ordered differently, they have to be reordered. Please note that a file in which the channel order is defined through the WAVEFORMATEXTENSIBLE_CHANNEL_MASK is not streamable, i.e. non-subset, as the tag is not found in each frame header. The mask bits can be found in the following table
+
+Bit number | Channel description
+----------:|:-----------
+0          | Front left
+1          | Front right
+2          | Front center
+3          | LFE
```

Comment:
As all the other channels are written out, I suggest to do the same here: `low-frequency effects`.